### PR TITLE
Categories

### PR DESCRIPTION
**Pull request by santiagogabrielcastillo**
The new `root_path `now is `/categories/:id` where, by default, the id is the Pastelería's id. 

image

### CODE REVIEW

File: app/views/categories/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<body class="bg-white">
+  <div class="container-fluid container-padding homepage">
+    <h4 class="ml-4 pl-2">SERVICIOS</h4>
+    <div class="align-content-center d-flex flex-wrap justify-content-center text-center">
+      <% @categories.each do |category| %>
+        <div class="col-4 col-lg-1 mt-4 mx-3 px-2 category">
+          <%= link_to category_path(category) do %>
+            <%= image_tag "#{category.name.downcase.gsub(" ", "-")}.png", class: "category-img" %>
```
Comment (jpbalarini) [inline app/views/categories/index.html.erb]: let's use single quotes for ruby code. Check the other files too

File: app/views/categories/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<body class="bg-white">
+  <div class="container-fluid container-padding homepage">
+    <h4 class="ml-4 pl-2">SERVICIOS</h4>
+    <div class="align-content-center d-flex flex-wrap justify-content-center text-center">
+      <% @categories.each do |category| %>
```
Comment (jpbalarini) [inline app/views/categories/index.html.erb]: cant' we use a partial for showing the categories?

File: app/views/categories/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<body class="bg-white">
```
Comment (jpbalarini) [inline app/views/categories/index.html.erb]: this view looks the same as the show

File: config/routes.rb [hunk 4]
```diff
@@ -2,7 +2,8 @@
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
   devise_for :users
-  root to: 'pages#home'
+  get '/', to: redirect('/categories/47')
```
Comment (jpbalarini) [inline config/routes.rb]: instead of doing a redirect, we could load this category on the index method and render the show view

**Review by jpbalarini — CHANGES_REQUESTED** [review]

**Comment by santiagogabrielcastillo** [comment]
Done all the tasks!

**Pull request merged**